src/client/cf.js

```diff
@@ -37,6 +37,17 @@ export default class CfApiClient {
     return user
   }
 
+  
+  async changePassword(username, password){
```

Comment:
```suggestion
  async changePassword(username, oldPassword, newPassword){
```

Reply:
If the user is using this, they likely don't know their old password

Reply:
And old password is optional in the UAA call since the caller (this service) has the admin flag set

Reply:
Hm that makes me think we maybe want to do different endpoints for change password and ForgotPassword, where forgot password would either give people a link to a change password page or set a random password & send it to them?

Reply:
I think we are fine here since by the time they see this form (and are provided the signed elements), we know they are who they say they are